### PR TITLE
ci: Simplify Gradle, prep for github actions & maven central

### PR DESCRIPTION
**Pull request by politeRaccoon**
- Simplifying Gradle 
- Preparing to replace build scripts for github actions
- Real goal is getting everything "in order" to publish to maven central using a github actions based workflow

### CODE REVIEW

**Comment by politeRaccoon** [comment]
No response on a Saturday night?  What's the world coming to?

**Comment by davidkun** [comment]
the irony is that I tried to review from my phone and then personal laptop, but something was weird. it wouldn't let me see this repo without authenticating with mitre's okta... so I gave up

File: src/main/java/org/mitre/caasd/commons/out/GzFileSink.java [hunk 1]
```diff
@@ -158,6 +158,7 @@ public class GzFileSink<T> implements Consumer<T>, Closeable {
      *                       are generated you'll open too many file handles and kill performance or
      *                       just crash.
      * @param expirationTime How long between writing pieces of data until a .gz file is closed
+     * @throws Gibberish
```
Comment (davidkun) [inline src/main/java/org/mitre/caasd/commons/out/GzFileSink.java:161]: 😆

File: scripts/ci/ci-script-colors.sh
Comment (davidkun) [inline scripts/ci/ci-script-colors.sh:1]: can't believe this `colors` script proliferated into `commons` too 🤦 

🪦 good riddance

**Review by davidkun — COMMENTED** [review]
nice PR 🙏

File: src/main/java/org/mitre/caasd/commons/out/GzFileSink.java (hunk 1 above)
Comment (politeRaccoon) [reply src/main/java/org/mitre/caasd/commons/out/GzFileSink.java:161]: oof.  I hope this got removed later.  It was add to check if a javadoc screwup made the build fail.  (which was true at some point)

**Pull request merged**